src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -416,6 +435,26 @@
         "uriBaseId": [ "uri" ]
       }
     },
+    "fileLocation": {
+      "description": "Specifies the location of a file.",
+      "additionalProperties": false,
+      "type": "object",
+      "properties": {
+
+        "uri": {
+          "description": "A string containing a valid relative or absolute URI.",
+          "type": "string",
+          "format": "uri"
+        },
+
+        "uriBaseId": {
+          "description": "A string which indirectly specifies the absolute URI with respect to which a relative URI in the \"uri\" property is interpreted.",
+          "type": "string"
+        }
+      },
+
+      "required": [ "uri" ]
+    },
     "fix": {
```

Comment:
@michaelcfanning Thank you for alphabetizing :-) And yes, in this intermediate state, you need `fileLocation` for `attachments` but it wouldn't be used anywhere else.

I suggest you do the changes in chronological order, since later changes often depend on earlier changes, as you saw here.

Reply:
@michaelcfanning @EasyRhin0 Also thanks for copying the definition, with its oh-so-carefully-crafted text :-), from the copy of the schema in the OASIS repo.